### ISSUE TITLE
Accessibility: Combobox: UI type is not read when combo boxes are accessed.  

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3450]
Description: UI type is not read when combo boxes are accessed.  

Expected:  Each & every UI should be correctly read based upon its type when accessed. 

Screenshot: 
![image](https://user-images.githubusercontent.com/32538291/94521733-2bc47f00-024c-11eb-8760-711f24409218.png)

**Comment by puru-hk** [comment]
Tested it is working as expected.
